data/json/items/tool/entry_tools.json

```diff
@@ -119,5 +119,26 @@
     "color": "light_gray",
     "qualities": [ [ "LOCKPICK", 40 ] ],
     "flags": [ "PSEUDO", "PERFECT_LOCKPICK" ]
+  },
+  {
+    "id": "heavy_crowbar",
+    "type": "TOOL",
+    "name": { "str": "heavy crowbar" },
+    "looks_like": "crowbar",
+    "description": "A 5 foot long construction prybar.  Essentially a long solid steel rod with a hardened chisel tip.",
+    "weight": "8 kg",
+    "to_hit": 1,
+    "price_postapoc": 4000,
+    "color": "red",
+    "symbol": "/",
+    "material": [ "steel" ],
+    "techniques": [ "WBLOCK_1", "SWEEP", "BRUTAL" ],
+    "flags": [ "DURABLE_MELEE", "ALWAYS_TWOHAND", "SHEATH_SPEAR" ],
+    "volume": "1 L",
+    "longest_side": "150 cm",
+    "bashing": 45,
```

Comment:
That's super heavy hitting. With a value like this one is able to crush brick walls and steel bar doors. Is this crowbar really that powerful?

Reply:
Yes, absolutely. Breaking concrete and ripping open doors that don't want to open are specifically two things it's really good at in real life. These are also values that I compared to other similar items in the game like the different sledge hammers and halligan bar.

You don't use it the same as a hammer. Instead of swinging it at something, you want to break, you stab it.